<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QFrame" name="frameToolbar">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>10</y>
     <width>781</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="buttonFullscreen">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>88</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Full Screen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSlideshow">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>88</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Slide Show</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttoPreferences">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>88</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Preferences</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frameBottom">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>379</y>
     <width>781</width>
     <height>91</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QToolButton" name="buttonPrevious">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>31</width>
      <height>91</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QToolButton" name="buttonNext">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>0</y>
      <width>31</width>
      <height>91</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QFrame" name="frameThumnailArea">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>721</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="viewArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>781</width>
     <height>311</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
